kyuubi-rest-client/pom.xml

```diff
@@ -33,13 +33,77 @@
 
     <properties>
         <jars.target.dir>${project.build.directory}</jars.target.dir>
+        <httpclient.version>4.5.13</httpclient.version>
+        <junit.version>4.13.2</junit.version>
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>commons-collections</groupId>
+            <artifactId>commons-collections</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>${httpclient.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client-runtime</artifactId>
+            <scope>runtime</scope>
```

Comment:
is it needed? @pan3793

Reply:
I think the client should work w/o hadoop-client-api/hadoop-client-runtime if user don't use kerberos/SPNEGO? If yes, I think we can mark these two dependencies as optional, and try catch the class not found exception to tip user add the hadoop dependencies if they are trying to use kerberos/SPNEGO